lms/djangoapps/verify_student/models.py

```diff
@@ -351,11 +351,22 @@ class PhotoVerification(IDVerificationAttempt):
     # capturing it so that we can later query for the common problems.
     error_code = models.CharField(blank=True, max_length=50)
 
+    # Mark if attempt is going through an A/B experiment
+    experiment_name = models.CharField(null=True, max_length=50)
```

Comment:
Is there prior art on tracking which bucket a user falls into as a database field like this? I didn't see any, but I might be searching for the wrong things.